### ISSUE TITLE
Allow variants of the same creature to be read as the same species by ASB.

### ISSUE THREAD

**Issue by soepie7** [cadon/ARKStatsExtractor#1194]
There are mods that allow conversion between normal, Aberrant, X, Tek, and what else there may be. This means that for the player, the line between those is rather blurry, unlike in the official game. It would be nice if you could for example tell ASB to just think of a Dodo and Aberrant Dodo as the same species, with perhaps only an additional variable stating which variant within that group it is.

**Comment by cadon** [comment]
You can change the species in ASB if you changed it ingame (select the creature(s) in the library, press F3 and select the species they should be converted to in the combobox on the bottm left).
I'm hesitant about adding an automatic species conversion. ARK doesn't allow breeding between different species, even if they're just variants, so keeping them separated is important for a breeding app. As I understood it, a species conversion happens rarely, e.g. if a mod is installed or uninstalled.

**Comment by soepie7** [comment]
I would agree that actual species conversion is a tricky thing. I meant it more than ASB would for example be able to make a group called "Rex" which contains the Rex, Aberrant Rex, and X-Rex.

It's also not just about the conversions being rare, but you may also want to easily compare stats and colors between the creatures of different variants without having to keep covnert them in ASB to their standard variant.

**Comment by coldino** [comment]
What you might not know - these species differ in more than just colour and name. For instance Aberrant variants have a different health and melee stat values compare at the vanilla ones, while X- variants have yet another health multiplier plus different level caps (and a lot more things beyond that). All of them have their own breeding times and preferred foods - important things that ASB needs to get right.

So the only option would be to somehow visually group then in the UI while keeping them separate under the hood. This is a surprisingly large amount of work and really I'm not sure how many users would ever use it.

**Comment by cadon** [comment]
If you just want to compare creatures, maybe the library filter is useful. In the library tab it's right of the settings/cog icon. If you enter `Rex` in there, it will display all creatures that have `Rex` in their species name, so it would list all variants of Rexes you have in your library (it will also display creatures that have Rex in another part of their properties, e.g. an Ankylosaur that is named Rex or owned by a player called Rex).
